### DISCUSSION TITLE
Hello World: Artisan on Mojave (Mac)?

### DISCUSSION THREAD

**Discussion by jwarthman** [artisan-roaster-scope/artisan#762] [Q&A] [answered]
My first post, and I’m ready to get started with Artisan controlling my Hottop B KN-8828B-2K+.

I’d like to run Artisan on my MacBookPro (Retina, Mid 2012) running Mojave (macOS 10.14.6).

But I see the in system requirements, Artisan. 2.4.6 requires macOS 10.15 and newer.

So my question, is this a “hard” requirement, or would macOS Mojave work? If not, is there an earlier Artisan version that would work with Mojave?

TIA,

Jim

**Comment by MAKOMO** [comment, marked as answer]
Well, a requirement is a requirement. Older versions are unsupported. macOS 10.14 can be updated on all hardware to macOS 10.15 at zero costs.

**Reply by jwarthman** [reply]
Thanks Marko, I appreciate your reply!  

In other instances a required system version can mean features break with prior system versions, or it can mean the software has not been tested with the older systems - so one might take a chance. In any case, I've resisted migrating beyond Mojave because certain (old) software will break, but maybe it's time.  :-)
